content/zh/docs/tasks/debug-application-cluster/get-shell-running-container.md

```diff
@@ -88,7 +88,7 @@ kubectl exec -it shell-demo -- /bin/bash
 The double dash symbol "--" is used to separate the arguments you want to pass to the command from the kubectl arguments.
 -->
 双破折号 "--" 用于将要传递给命令的参数与 kubectl 的参数分开。
-note >}}
+{{< /note >}}
```

Comment:
pls also remove line 2-4.
This PR is automatically adding the English reviewer given those lines.